meta/main.yml

```diff
@@ -21,4 +21,5 @@ galaxy_info:
       versions:
         - all
 
+allow_duplicates: true
```

Comment:
Typo in the commit message: evauated -> evaluated

Reply:
Whoops, fixed in 47226ac.